app/views/curation_concerns/base/_chf_show_page.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="work-show-page-thumb">
```

Comment:
not sure I understand this partial. It doesn't seem like it's the representative image, judging from comment in `_show_page_image`. Maybe rename this to distinguish from that one? and/or add a comment explaining what it's about?